packages/frontend/i18n/src/resources/en.json

```diff
@@ -1406,6 +1406,7 @@
   "com.affine.share-page.footer.description": "Empower your sharing with AFFiNE Cloud: One-click doc sharing",
   "com.affine.share-page.footer.get-started": "Get started for free",
   "com.affine.share-page.header.present": "Present",
+  "com.affine.share-page.header.login": "Login Or Sign Up",
```

Comment:
"Login or Sign Up"

Reply:
by design

Reply:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/T2klNLEk0wxLh4NRDzhk/c837aaea-284a-4e3f-a141-78897f1a3e26.png)

Reply:
I'm not sure what you mean here. Incorrect grammar in the product looks unprofessional.

Reply:
@qpomelo Can you explain this?

Reply:
Based on our internal discussion about button styling, each word within buttons was initially capitalized. However, I suggest we refine our rules to exclude prepositions, conjunctions, and similar words from capitalization.